zebra-chain/src/sapling/keys.rs

```diff
@@ -5,8 +5,8 @@
 //! diversified payment address addr_d = (d, pk_d ) are derived from sk,
 //! as described in [Sapling Key Components][ps]." - [§3.1][3.1]
 //!
-//! [ps]: https://zips.z.cash/protocol/protocol.pdf#saplingkeycomponents
-//! [3.1]: https://zips.z.cash/protocol/protocol.pdf#addressesandkeys
+//! [ps]: <https://zips.z.cash/protocol/protocol.pdf#saplingkeycomponents>
+//! [3.1]: <https://zips.z.cash/protocol/protocol.pdf#addressesandkeys>
```

Comment:
Can you also confirm if this ones also produces warnings. I think the `<` and `>` purpose is to make the links clickeable however this ones are not displayed as `http://`.

Check for example https://doc.zebra.zfnd.org/zebra_chain/sapling/keys/index.html

Thanks.

Reply:
This is a good point. That doesn't produce a warning. However, it is not wrong to have the angle brackets there either. See here https://www.markdownguide.org/basic-syntax/#formatting-the-second-part-of-the-link. Angle brackets are optional in this case. It think it might be good to use them here as well for consistency.

Reply:
I don't have a strong opinion but i slightly prefer to don't add the extra characters if they are not needed. I see the extra characters as a way to make the link clickable and not that much as the correct format. In this context making them clickable don't have much sense as they are not visible.

Reply:
Resolved in [8b091aa](https://github.com/ZcashFoundation/zebra/pull/4494/commits/8b091aa9fab453e7d3559a5d474e0879183b9bfb).

Reply:
Actually, the references also require the angle brackets because `cargo doc` does produce warnings if the angle brackets are missing. I'm going to revert the reversion.

Reply:
Hmm, so it seems the lint checks every link inside rustdoc comment whatever is visible or not. I think this might be a bug in the lint or they will have reasons. Anyway, if it warns we should do the change. Thanks.

Reply:
See https://github.com/ZcashFoundation/zebra/pull/4494#discussion_r882650766. `cargo doc` doesn't actually produce warnings in this case.

Reply:
Thank you, that makes much more sense to me.